src/libraries/System.Private.Xml/src/System/Xml/XmlCharType.cs

```diff
@@ -283,46 +228,4106 @@ private static bool InRange(int value, int start, int end)
             return (uint)(value - start) <= (uint)(end - start);
         }
 
-#if XMLCHARTYPE_GEN_RESOURCE
-        //
-        // Code for generating XmlCharType.bin table and s_PublicIdBitmap
-        //
-        // build command line:  csc XmlCharType.cs /d:XMLCHARTYPE_GEN_RESOURCE
-        //
-        public static void Main( string[] args ) {
-            try {
-                InitInstance();
-
-                // generate PublicId bitmap
-                ushort[] bitmap = new ushort[0x80 >> 4];
-                for (int i = 0; i < s_PublicID.Length; i += 2) {
-                    for (int j = s_PublicID[i], last = s_PublicID[i + 1]; j <= last; j++) {
-                        bitmap[j >> 4] |= (ushort)(1 << (j & 0xF));
-                    }
-                }
-
-                Console.Write("private const string s_PublicIdBitmap = \"");
-                for (int i = 0; i < bitmap.Length; i++) {
-                    Console.Write("\\u{0:x4}", bitmap[i]);
-                }
-                Console.WriteLine("\";");
-                Console.WriteLine();
-
-                string fileName = ( args.Length == 0 ) ? "XmlCharType.bin" : args[0];
-                Console.Write( "Writing XmlCharType character properties to {0}...", fileName );
+        private static ReadOnlySpan<byte> CharProperties => new byte[]
```

Comment:
why ``=>`` and not just ``=`` ?
also should be readonly

Reply:
=> not = because it's a property accessor function not a field assignment and it can't be readonly because span fields can only be declared in ref-only structs which this is not. 

It might be possible to do Unsafe things to take the ROS from the binary section and wrap it in a ROM somehow but doing that would come with overhead and possibly return the locking of the original and at that point we aren't making progress we're just rearranging deckchairs.